scene/3d/camera_3d.cpp

```diff
@@ -687,6 +769,9 @@ RID Camera3D::get_pyramid_shape_rid() {
 Camera3D::Camera3D() {
 	camera = RenderingServer::get_singleton()->camera_create();
 	set_perspective(75.0, 0.05, 4000.0);
+	oblique_normal = Vector3();
```

Comment:
Initialize to Vector3(0, 0, -1) and don't initialize twice in the initializer and the constructor.